about.html

```diff
@@ -0,0 +1,111 @@
+         to expand the knowledge of high school students in architecture, hone their skills in 
+         design and prepare them for their careers. Led by a team of seasoned professionals, 
+         the bootcamp covers a wide range of topics, from design principles and construction 
+         techniques to project management and sustainability.<br><br>
+         Over the course of the program, participants will work on a variety of real-world projects, 
+         both individually and in teams. They will learn how to create detailed architectural plans, 
+         develop 3D models, and use cutting-edge software to bring their designs to life. In addition, 
+         they will gain hands-on experience with materials and construction methods, visiting 
+         construction sites and participating in workshops led by industry experts.</p>
+      <p class="send-email">Please contact us per Email for any further questions about TIACE Young Architects Bootcamp 2024!<br>
+        <a href="mailto:bootcamp2023@tiace.com">bootcamp2023@tiace.com</a></p>
+    </section>
+    <section id="logo-design">
+      <h2>Tashkent Institute of Architecture and Civil Engineering Architecture Bootcamp 2021 Logo</h2>
+      <hr>
+      <p>The Logo of TIACE Architecture Bootcamps 2021 was decided through
+        a logo competition from 12.June to 24.July</p>
+      <img src="./icons/logo2.png">
+    </section>
+    <hr class="grey-wide">
+    <section id="past-bootcamps">
+      <h2>See the past TIACE Architecture Bootcamps</h2>
+      <hr>
+      <p>Take a look at the last three TIACE Architecture Bootcamps and read the stories of our students</p>
+      <ul>
+        <li><a href="#">TIACE Architecture Bootcamp <h3>2023</h3></a></li>
+        <li><a href="#">TIACE Architecture Bootcamp <h3>2022</h3></a></li>
+        <li><a href="#">TIACE Architecture Bootcamp <h3>2021</h3></a></li>
+      </ul>
+    </section>
+    <section id="partners">
+      <h2>Partner</h2>
+      <hr>
+      <ul class="partners-list">
+        <li><a href="https://tcibc.uz/">TASHKENT CITY</a></li>
+        <li><a href="https://www.instagram.com/academic_uz/">ACADEMIC</a></li>
+        <li><a href="https://it-park.uz/">IT PARK</a></li>
+        <li><a href="https://newuu.uz/en">NEW UZBEKISTAN UNIVERSITY</a></li>
+        <li><a href="https://mc.uz/?lang=ru">Ministry of Construction<br> of the Republic of Uzbekistan</a></li>
+      </ul>
+    </section>
+  </main>
+  <footer class="latto about">
+    <img class="footer-desktop-img" src="./icons/logo3.png" alt="Logo">
+    <div>
+      <p>@ 2024 Tashkent Institute of Architecture and Civil Engineering. All Rights Reserved.</p>
+      <p class="privacy-policy hidden">For details about how we use your information, please see our <a href="#">Privacy Policy</a>.</p>
+      <p class="terms-cookies hidden"><a href="#">Terms & Conditions</a> | <a href="#">Cookies</a></p>
+    </div>
+  </footer>
```

Comment:
Same here. Kindly align the footer contents in the center just like in the provided design.
![image](https://github.com/Zilola-Nazarova/capstone-project-module-1/assets/33268927/47550ad3-d3aa-4691-9dfa-9465f641a0aa)